src/components/layout/progress/PageLoader/PageLoader.tsx

```diff
@@ -13,7 +13,9 @@ function PageLoader({
         className="flex flex-column center white"
         style={{
           padding: '30px',
-          width: 'fit-content',
+          width: '30%',
+          minWidth: '300px',
+          wordBreak: 'break-word',
```

Comment:
this loading modal would get bigger and smaller with the text. it was annoying with the callback messages having various lengths.